src/gimelstudio/core/node/property.py

```diff
@@ -144,6 +145,29 @@ def CreateUI(self, parent, sizer):
         self.AddToFoldPanel(sizer, fold_panel, self.img)
 
 
+class ColorProp(Property):
+    """ 
+    Allows the user to select a color.
+    """
+    def __init__(self, idname, default=(255, 255, 255), visible=True):
```

Comment:
1. It looks like you're missing the ``fpb_label="", expanded=True`` variables here which the Property init expects (in line 153).

2. I would add the alpha value to the ``default`` variable since the colorpicker returns an RGBA value. e.g: ``(255, 255, 255, 255)``